File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

void block(int n);
void spaces(int n);


int main(void)
{
    // Asks user for positive integer between 1 and 8
    int levels;
    do
    {
        levels = get_int("Height: ");
    }
    while (levels < 1 || levels > 8);
    
    // Creates pyramid with levels indicated by user
    int spacemove = 1;
    for (int i = 0; i < levels; i++)
    {
        spaces(levels - spacemove);
        spacemove++;
        block(i + 1);
        spaces(2);
        block(i + 1);
        printf("\n");
    }
}

// Function that prints blocks
void block(int n)
{
    for (int j = 0; j < n; j++)
    {
        printf("#");
    }
}

// Function that prints spaces
void spaces(int n)
{
    for (int u = 0; u < n; u++)
    {
        printf(" ");
    }
}

File: pset6/mario/more/mario.py
import csv
import sys
import random
import cs50
from cs50 import get_int

size = get_int("Height: ")

while(size > 8 or size < 1):
    print("Pyramid must be between 1 and 8 blocks high")
    size = get_int("Height: ")

spaces = size - 1
for i in range(1, size + 1, 1):
    for j in range(spaces):
        print(" ", end="")
    print("#" * i, end="")
    print("  ", end="")
    print("#" * i)
    spaces -= 1

